### PR TITLE
updated al-collector-js package to 3.0.8 and fixed testcases

### PR DESCRIPTION
**Pull request by imranalisyed506**
### Problem Description
update al-collector-js package to 3.0.8 and fix testcases
### Solution Description
updated al-collector-js package to 3.0.8 and fix testcases

request library is deprecated replace with axios library npm package
"request": "^2.88.2",
"request-promise-native": "^1.0.9"

### CODE REVIEW

File: test/al_aws_collector_test.js [hunk 1]
```diff
@@ -803,7 +803,7 @@ describe('al_aws_collector tests', function() {
 
         it('send logmsgs got failed', function(done) {
                 ingestCLogmsgsStub.restore();
-                let logmsgErr = {"errorType":"StatusCodeError","errorMessage":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","name":"StatusCodeError","statusCode":400,"message":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","error":"{\"error\":\"body encoding invalid\"}","options":{"method":"POST","url":"https://api.global-services.us-west-2.global.alertlogic.com/ingest/v1/48649/data/logmsgs"}};
+                let logmsgErr = {"errorType":"StatusCodeError","errorMessage":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","name":"StatusCodeError","statusCode":400,"message":"400 - \"{\\\"error\\\":\\\"body encoding invalid\\\"}\"","error":"{\"error\":\"body encoding invalid\"}","response":{"status": 400 },"options":{"method":"POST","url":"https://api.global-services.us-west-2.global.alertlogic.com/ingest/v1/48649/data/logmsgs"}};
```
Comment (scott-jones-AlertLogic) [inline test/al_aws_collector_test.js:806]: just for completeness, should we remove `logmsgErr.statusCode` from the log error now that it's replaced with `logmsgErr.response.status`?

Comment (imranalisyed506) [reply test/al_aws_collector_test.js]: removed logmsgErr.statusCode for completeness

**Review by scott-jones-AlertLogic — APPROVED** [review]
super, thanks Imran!

**Review by rakhimundhada15 — APPROVED** [review]
LGTM

**Pull request merged**